libvmi/read.c

```diff
@@ -97,13 +97,13 @@ vmi_read(
 
         if(dtb) {
             paddr = vmi_pagetable_lookup(vmi, dtb, start_addr + buf_offset);
+            if (!paddr) {
```

Comment:
+1, that's reasonable, let's just open an issue for this to not forget that v2p could result in p=0.